app/scripts/components/common/chart/index.tsx

```diff
@@ -199,13 +221,32 @@ function RLineChart(props: RLineChartProps, ref: RefObject<HTMLDivElement>) {
             />
           )}
           {renderBrush &&
-            renderBrushComponent({
-              chartData,
-              xKey,
-              uniqueKeys,
-              lineColors,
-              dateFormat
-            })}
+            <Brush
+              data={chartData}
+              dataKey={xKey}
+              height={brushHeight}
+              tickFormatter={(t) => timeFormatter(t, dateFormat)}
+              onChange={handleBrushChange}
+              startIndex={brushStartIndex}
```

Comment:
I thought setting index with state value will fix Brush Handle problem but it doesn't. :/ 
I looked up the repo and this bug is supposed to be fixed: https://github.com/recharts/recharts/issues/1796 but I still see the problem (ex. simplified demo that I copied and tweaked: https://codesandbox.io/s/rechart-brush-sync-z9bk04) I will file an issue, but not sure if I can do anything more about it.